htword/WordKey.cc

```diff
@@ -369,8 +369,9 @@ WordKey::Prefix() const
     //
     // Fields set, then fields unset then field set -> not a prefix
     //
-    if(IsDefined(j))
+    if(IsDefined(j)) {
```

Comment:
We're going with the GNU style formatting (see the note in #documentation), which means the brace  should be moved to the next line after the if statement.

https://www.gnu.org/prep/standards/html_node/Formatting.html

I know the rest of the code doesn't match yet. Kind of a pain, but keeping everything uniform makes things easier and more efficient for collaborators in the long-term.